docs/pages/admin-guides/management/guides/awsoidc-integration-rds.mdx

```diff
@@ -0,0 +1,289 @@
+```yaml
+kind: role
+version: v7
+metadata:
+  name: example
+spec:
+  allow:
+    rules:
+    - resources:
+      - integration
+      verbs:
+      - create
+      - update
+      - list
+      - read
+```
+
+## Step 2/2. Visit the Web UI
+
+The RDS enrollment wizard is available in the "Add new resource" panel of the
+Teleport Web UI:
+
+![Screenshot of RDS enrollment wizard tile](../../../../img/database-access/enrollment-wizard/resource-panel-rds.png)
+
+The Teleport Web UI walks you through the steps to set up an AWS OIDC
+integration (if you don't have one already), configure IAM permissions, create
+an ECS deployment in your AWS account, and run a database connection test.
+
+## How it works
+
+The RDS enrollment wizard depends on the [AWS OIDC integration](./awsoidc-integration.mdx).
+The AWS OIDC integration creates and configures an AWS IAM OpenID Connect
+Identity Provider (OIDC IdP) and an AWS IAM role that your Teleport cluster can
+assume.
+
+The RDS enrollment wizard adds permissions to the AWS OIDC integration IAM role
+to list RDS databases and VPC resources like subnets and security groups and
+to deploy an AWS ECS service.
+
+Once the integration role permissions are configured, the wizard walks you
+through deploying the Teleport database service in a high availability (HA)
+setup on AWS Fargate for AWS ECS.
+
+First, it creates an IAM role for Teleport database service instances to use.
+This role has permissions to output logs in ECS, describe RDS databases, and
+connect to RDS databases.
+
+Next, it creates an ECS task definition that includes various settings for
+configuring and starting a Teleport database service instance. 
+The task definition uses the IAM role created in the first step as both the
+[ECS task execution role](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_execution_IAM_role.html).
+and the
+[ECS task role](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html)
+
+The wizard then creates an ECS deployment in the same AWS VPC as your RDS
+databases with two copies of the Teleport database service instance task for
+an HA setup.
+By deploying the Teleport database service instances in the same VPC as the RDS
+databases, the instances can connect to databases in private subnets.
+
+Once an ECS task starts, the Teleport database service instance establishes
+a tunnel connection to your Teleport cluster.
+
+This is a serverless HA approach to deploying the Teleport database service.
+It does not require any allowed inbound traffic from the internet and supports
+[automatic agent upgrades](../../../upgrading/automatic-agent-updates.mdx).
+
+The Teleport database service instances use
+[dynamic database registration](../../../enroll-resources/database-access/guides/dynamic-registration.mdx)
+so that you can enroll or disenroll databases without restarting or modifying
+the deployment.
+The instances will match and proxy connections for all dynamic `db` resources in
```

Comment:
did we mention how the dynamic `db` gets added or did i miss that?

on the same topic, did we mention how discovery service play a role in this flow (for auto-discovery)?

Reply:
> did we mention how the dynamic db gets added or did i miss that?

I missed that. I'll add that info

> on the same topic, did we mention how discovery service play a role in this flow (for auto-discovery)?

I've been kind of pretending like the auto-discovery option doesn't exist because it complicates things so much more lol.
I can add that as well though.

Reply:
updated to explain the `db` objects in 2b316271e5fd9496db4c106da93de4e45005f1c6

I didn't go in depth about the discovery service's role though